examples/webgl_modifier_subdivision.html

```diff
@@ -70,6 +74,15 @@
 				controls.target.set( 0, 0, 0 );
 				controls.update();
 
+				scene.add( new THREE.AmbientLight( 0xffffff, 0.25 ) );
+				scene.add( new THREE.HemisphereLight( 0xffffff, 0x444444, 0.75 ) );
```

Comment:
Nit: It is odd to add both of these indirect light sources. This is equivalent.

```js
scene.add( new THREE.HemisphereLight( 0xffffff, 0x737373, 1 ) );
```

Reply:
Okay, I have removed the `AmbientLight`.